<launch>

    <group ns="cairo_go1">
        
        <!-- Go1 Motion Planner (goals -> waypoints) -->
        <node name="dummy_planner" pkg="medos_demo" type="go1_dummy_path_planner.py" output="screen">
            <rosparam file="$(find medos_demo)/maps/go1_dummy_paths.yaml" command="load" />
        </node>
        
        <!-- High level controller for go1 (Recap commands -> all the other nodes) -->
        <node name="go1_high_level_controller" pkg="medos_demo" type="high_level_controller.py" output="screen">
            <param name="agent_type" value="SCIENTIST" />
        </node>

    </group>

</launch>